docs: fix linter errors for workspace-config.md (#47709)
angular/angular: aio/content/guide/workspace-config.md

| Property      | Details |
|:---           |:---     |
| `root`        | The root directory for this project's files, relative to the workspace directory. Empty for the initial application, which resides at the top level of the workspace. |
| `sourceRoot`  | The root directory for this project's source files.                                                                                                        |

> | `projectType` | One of "application" or "library." An application can run independently in a browser, while a library cannot.                                           |

Reviewer: ```suggestion
| `projectType` | One of "application" or "library" An application can run independently in a browser, while a library cannot.                                           |
```

Author: Done.

|:---              |:---     |
| `glob`           | A [node-glob](https://github.com/isaacs/node-glob/blob/master/README.md) using `input` as base directory.                                                              |
| `input`          | A path relative to the workspace root.                                                                                                                                 |

> | `output`         | A path relative to `outDir` \(default is `dist/` *project-name*\). Because of the security implications, the Angular CLI never writes files outside of the project output path. |

Reviewer: ```suggestion
| `output`         | A path relative to `outDir` \(default is `dist/`*project-name*\). Because of the security implications, the Angular CLI never writes files outside of the project output path. |
```

Author: Hi @alan-agius4,

Reverting this change will give the "The code term should be followed by a space or punctuation." error, which can't be disabled. 
It would work if dist is changed to a regular word instead of a code term.

</code-example>

> The contents of `node_modules/some-package/images/` are available in `dist/some-package/`.

Reviewer: This change changes the context. `dist/some-package/` will be available in the future after the build. 

```suggestion
The contents of `node_modules/some-package/images/` will be available in `dist/some-package/`.
```

Author: Done.
